src/pages/static/CookiesPolicy.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import config from 'config/constants';
+
+function CookiesPolicy() {
```

Comment:
Please add a @TODO here, mentioning that this is only a temporal approach. Later on we want to use markdown.

Reply:
ok!! This evening I'll do everything

Reply:
This has been done, I don't know why it's still here the comment